model-optimizer/mo/utils/ir_reader/restore_graph.py

```diff
@@ -64,7 +90,6 @@ def save_restored_graph(graph: Graph, path: str, meta_data, name=None):
         PackBinaryWeights,
         BlobNormalizer,
         ConvolutionNormalizer,
-        KaldiRemoveMemoryOutputBackReplacementPattern,
```

Comment:
@sadolini, could you please tell if this affects Kaldi models because above we have a comment about "do not change"?

Reply:
It is my comment and it about list elements order, not about removing or adding elements.